index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# website: <<https://URL>>"></head>
```

Comment:
```
<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# website: <<https://URL>>">
```

<>

Reply:
```suggestion
<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# website: <<https://URL>>"></head>
```
うわ・・・閉じタグ忘れてました
ご指摘ありがとうございます◎

Reply:
```suggestion
<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# website: <<https://URL>>">
```